porcupine/plugins/sidebar.py

```diff
@@ -0,0 +1,54 @@
+import pathlib
+import tkinter
+import tkinter.ttk as ttk
+from typing import Dict, List
+
+from porcupine import get_paned_window, get_tab_manager, tabs
+
+
+class Sidebar(ttk.Treeview):
+
+    def __init__(self, master: tkinter.Misc) -> None:
+        super().__init__(master, selectmode='browse')
+        self.nodes: Dict[str, pathlib.Path] = {}
+        self.nodes[''] = pathlib.Path('.').resolve()
```

Comment:
```suggestion
        self.nodes = {'': pathlib.Path('.').resolve()}
```

Maybe this should also be renamed to `paths`, because the values are now paths?